step-cli/installation.mdx

```diff
@@ -176,9 +176,9 @@ Download the Windows binary from our [latest releases](https://github.com/smalls
 Open PowerShell and run the following:
 
 ```shell
-curl.exe -LO https://dl.smallstep.com/gh-release/cli/docs-cli-install/v0.24.4/step_windows_0.24.4_amd64.zip
-Expand-Archive -LiteralPath .\step_windows_0.24.4_amd64.zip -DestinationPath .
-step_0.24.4\bin\step.exe version
+curl.exe -LO https://dl.smallstep.com/cli/docs-cli-install/latest/step_windows_amd64.zip
+Expand-Archive -LiteralPath .\step_windows_amd64.zip -DestinationPath .
+step_0.25.0\bin\step.exe version
```

Comment:
Is this final version number in the path intentional?

Reply:
It is intentional but on second though we should probably use this opportunity to remove the version number from the wrapped dirname. I can't see why we wouldn't do that. 
So I created these two PRs to do that:
https://github.com/smallstep/cli/pull/1034
https://github.com/smallstep/certificates/pull/1580

If we agree that this makes sense, I will hold this PR open until the next release.